guides/v2.0/cloud/configure/cloud-vcl-custom-snippets.md

```diff
@@ -87,244 +93,155 @@ The following is an example of a returned JSON for a customer VCL snippet:
-* `type`: Specifies the location to place the generated snippet such as `init` (above subroutines) and within subroutines like `recv`. See [Fastly VCL snippet object values](https://docs.fastly.com/api/config#snippet){:target="_blank"} for information on these values.
-* `content`: The snippet of VCL code to run in one line, without line breaks.
+-   `name`—Name for the VCL snippet.
+-   `dynamic`—Indicates if this is a <a href="https://docs.fastly.com/guides/vcl-snippets/using-dynamic-vcl-snippets">dynamic snippet</a> or <a href="https://docs.fastly.com/guides/vcl-snippets/using-regular-vcl-snippets">regular snippet</a>.
+-   `type`—Specifies the location to place the generated snippet such as `init` (above subroutines) and within subroutines like `recv`. See [Fastly VCL snippet object values](https://docs.fastly.com/api/config#snippet){:target="\_blank"} for information on these values.
+-   `priority`—Determines the order VCL snippets call. Lower values run first, from 1 to 100. All Magento module uploaded snippets are 50. If you want an action to occur last or override Magento default VCL snippets, enter a higher number like 100. To have code occur immediately, enter a lower value like 5.
+-   `content`—The snippet of VCL code to run in one line, without line breaks.
 
 For detailed examples and custom code, see the following:
 
-* [Custom whitelist VCL]({{page.baseurl}}cloud/configure/fastly-vcl-whitelist.html)
-* [Custom blacklist VCL]({{page.baseurl}}cloud/configure/fastly-vcl-blacklist.html)
-* [Custom extend Admin timeout VCL]({{page.baseurl}}cloud/configure/fastly-vcl-extend-timeout.html)
-* [Custom redirect to Wordpress VCL]({{page.baseurl}}cloud/configure/fastly-vcl-wordpress.html)
-* [Custom block bad referer VCL]({{page.baseurl}}cloud/configure/fastly-vcl-badreferer.html)
+-   [Custom whitelist VCL]({{page.baseurl}}cloud/configure/fastly-vcl-whitelist.html)
+-   [Custom blacklist VCL]({{page.baseurl}}cloud/configure/fastly-vcl-blacklist.html)
+-   [Custom extend Admin timeout VCL]({{page.baseurl}}cloud/configure/fastly-vcl-extend-timeout.html)
+-   [Custom redirect to Wordpress VCL]({{page.baseurl}}cloud/configure/fastly-vcl-wordpress.html)
+-   [Custom block bad referer VCL]({{page.baseurl}}cloud/configure/fastly-vcl-badreferer.html)
 
 <!-- ### Update an existing VCL snippet {#update}
 Locate the snippet you want to update from the list of snippets included in the cloned version. You can use the following command to list the snippets:
 
-	curl -X GET -s https://api.fastly.com/service/<Service ID>/version/<Cloned version #>/snippet/ -H "Fastly-Key:FASTLY_API_TOKEN"
+	curl -X GET -s https://api.fastly.com/service/<FASTLY_SERVICE_ID>/version/<Cloned version #>/snippet/ -H "Fastly-Key: <FASTLY_API_TOKEN>"
```

Comment:
You've made updates to a section that is commented out; starting on line 172. Should we uncomment this section? If not, I recommend deleting it.

Reply:
No, we don't need to uncomment it. We can remove this section.